src/examples/shared/useCalculateScroll.ts

```diff
@@ -0,0 +1,61 @@
+import { useLayoutEffect, useState } from "react"
+import analyticsEventSections from "./example-sections/constants"
+import { majorScale } from "evergreen-ui"
+
+const clamp = (value: number) => Math.max(0, value)
```

Comment:
Would these two functions be obsolete with [`clamp`](https://lodash.com/docs/4.17.15#clamp) and [`inRange`](https://lodash.com/docs/4.17.15#inRange)?

Reply:
Ah I didn't realize this was a lodash function too